java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/JUnit5Runner.java

```diff
@@ -18,6 +20,8 @@ public class JUnit5Runner {
 
   public static void main(String[] args) {
     var testSuite = System.getProperty("bazel.test_suite");
+    List<String> includeEngines = System.getProperty("bazel.include_engines") == null ? null : List.of(System.getProperty("bazel.include_engines").split(","));
```

Comment:
Please could you move the extraction of the runner-specific values to the `ActualRunner`? That way if we add support for (eg) TestNG or JUnit 6 (should it ever exist) we'll have all the logic localised in one place.